core/serialisation/data_serialiser.py

```diff
@@ -0,0 +1,333 @@
+    state = auto_field(data_key="ActualOrForecast")
+    amount = auto_field(data_key="Amount")
+    additional_information = auto_field(data_key="AdditionalInformation")
+    project_name = auto_field(model=Project, data_key="ProjectName")
+    programme_name = auto_field(model=Programme, data_key="Place")
+    organisation_name = auto_field(model=Organisation, data_key="OrganisationName")
+
+
+class OutputDimSchema(SQLAlchemySchema):
+    """Serialise OutputDim Reference data."""
+
+    class Meta:
+        model = OutputDim
+
+    output_name = auto_field(data_key="OutputName")
+    output_category = auto_field(data_key="OutputCategory")
+
+
+class PlaceDetailSchema(SQLAlchemySchema):
+    """Serialise PlaceDetail data."""
+
+    class Meta:
+        model = PlaceDetail
+
+    submission_id = auto_field(model=Submission, data_key="SubmissionID")
+    programme_id = auto_field(model=Programme, data_key="ProgrammeID")
+    question = auto_field(data_key="Question")
+    indicator = auto_field(data_key="Indicator")
+    answer = auto_field(data_key="Answer")
+    organisation_name = auto_field(model=Organisation, data_key="OrganisationName")
+
+
+class PrivateInvestmentSchema(SQLAlchemySchema):
+    """Serialise PrivateInvestment data."""
+
+    class Meta:
+        model = PrivateInvestment
+
+    submission_id = auto_field(model=Submission, data_key="SubmissionID")
+    project_id = auto_field(model=Project, data_key="ProjectID")
+    total_project_value = auto_field(data_key="TotalProjectValue")
+    townsfund_funding = auto_field(data_key="TownsfundFunding")
+    private_sector_funding_required = auto_field(data_key="PrivateSectorFundingRequired")
+    private_sector_funding_secured = auto_field(data_key="PrivateSectorFundingSecured")
+    additional_comments = auto_field(data_key="PSIAdditionalComments")
+    project_name = auto_field(model=Project, data_key="ProjectName")
+    programme_name = auto_field(model=Programme, data_key="Place")
+    organisation_name = auto_field(model=Organisation, data_key="OrganisationName")
+
+
+class ProgrammeProgressSchema(SQLAlchemySchema):
+    """Serialise ProgrammeProgress data"""
+
+    class Meta:
+        model = ProgrammeProgress
+
+    submission_id = auto_field(model=Submission, data_key="SubmissionID")
+    programme_id = auto_field(model=Programme, data_key="ProgrammeID")
+    question = auto_field(data_key="Question")
+    answer = auto_field(data_key="Answer")
+    programme_name = auto_field(model=Programme, data_key="Place")
+    organisation_name = auto_field(model=Organisation, data_key="OrganisationName")
+
+
+class ProgrammeSchema(SQLAlchemySchema):
+    """Serialise Programme data."""
+
+    class Meta:
+        model = Programme
+
+    programme_id = auto_field(data_key="ProgrammeID")
+    programme_name = auto_field(data_key="ProgrammeName")
+    fund_type_id = auto_field(data_key="FundTypeID")
+    organisation_name = auto_field(model=Organisation, data_key="OrganisationName")
```

Comment:
I get why Place isn't included here but is this going to confuse users if they're expecting it as "Place"? Seems mental adding it in twice I know..

Reply:
oh, it's not just me, SQLAlchemy also thinks it's not a good option, no super obvious option to duplicate the column. I'll hack it...

Reply:
hmm, no actual way to do this cleanly, so I'm going to reject it, as unreasonable and un-programmatic. sqlalchemy-marshmallow will not allow the same field twice, the second overwrites the first. If anyone complains we'll 1) explain why we shouldn't, and 2) if necessary, import base marshmallow field class, and add an aliased column to the query - but feels very hacky

Reply:
makes sense